python/tank/api.py

```diff
@@ -798,8 +799,76 @@ def preview_filesystem_structure(self, entity_type, entity_id, engine=None):
 
 def sgtk_from_path(path):
     """
-    Creates a Toolkit Core API instance based on a path inside a project
-    or path pointing directly at a pipeline configuration.
+    Creates a Toolkit Core API instance based on a path to a configuration
```

Comment:
note: next up for me is to write docs for bootstrapping - i will do a big pass on the sphinx init docs as part of that, to make sure that we have a decent outline and structure etc.